jobs/telnyx-python-developer.html

```diff
@@ -0,0 +1,77 @@
+---
+title: Python Develooer 
+company: Telnyx
+url: https://telnyx.com/company/careers
+location: Dublin, Ireland + Warsaw, Poland +Chicago, USA
+          Permanent
+contact:
+    name: Katie Glendon
+    email: katie@telnyx.com
+    phone:
+    
+created: !!timestamp 2019-07-11
+tags:
+  - python
+  - sql
+  - ansible
+  - docker
+  - rabbitmq
+  - django
+  # include relevant tags
+---
+
+# Python Developer
+Our Python team work on building highly available low latency microservices from scratch using the newest bleeding edge technologies. We are looking for someone who will work within our awesome Python team, championing and leading the build of secure, robust and high-performance systems. 
+
+You will work on creating internal tools to be used across the company, have the opportunity to make a huge impact on our growing company, solving complex problems as we scale. You’ll be part of a group of people working together to build solutions to mission-critical problems and a company that values the very best ideas. People rely on our products to communicate daily, which means they rely on us to build things with a high degree of resiliency and reliability.
+
+## In This Role You Will:
+
+
+Design developer-friendly APIs.
+
+Build, test, and deploy applications to deliver critical global communications services.
+
+Identify and resolve issues related to latency, scalability and performance.
+
+Create tools to automate aspects of production systems.
+
+
+
+
+##You May Be A Fit If You Have:
+
+3+ years of professional software development experience with Python
```

Comment:
We advise against specifying a number of years of experience. We find that there are young developers who are excellent, and sometimes very experienced developers who aren't so good. You should list the qualities you want from a developer and avoid anything to do with their background.